ts/src/pro/ascendex.ts

```diff
@@ -222,23 +230,26 @@ export default class ascendex extends ascendexRest {
         params = this.extend (params, {
             'ch': channel,
         });
-        const orderbook = await this.watchPublic (channel, params);
+        const method = this.handleOrderBookSubscription;
+        const orderbook = await this.watchPublic (channel, symbol, method, params);
         return orderbook.limit ();
     }
 
-    async watchOrderBookSnapshot (symbol: string, limit: Int = undefined, params = {}) {
+    async watchOrderBookSnapshot (client, message, subscription) {
```

Comment:
here too, changed signature as it's on all other exchanges for this method (ofc, changed arguments from everywhere this method is being called)